### PR TITLE
Update go-autorest version v7.0.5

### PR DESCRIPTION
**Pull request by garimakhulbe**

### CODE REVIEW

**Comment by ahmetalpbalkan** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ go: 1.6
 install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
+  - go get -u golang.org/x/net/context
   - export GO15VENDOREXPERIMENT=1
```
Comment (garimakhulbe) [inline .travis.yml]: @ahmetalpbalkan Also added this.

Comment (ahmetalpbalkan) [inline .travis.yml]: @garimakhulbe hmm what purpose does it serve? Do any of the libs have any dependencies on this?

Comment (garimakhulbe) [inline .travis.yml]: We are using crypto which has dependency in this and build was failing due to this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 -----
 
+## `v2.2.0-beta`
+
+- Uses go-autorest version v7.0.5.
+- Update packages jwt-go and crypto in go-autorest and SDK.
```
Comment (ahmetalpbalkan) [inline CHANGELOG.md]: this is a bit cryptic. You should probably say `Updated the versions of packages "a" and "b" in glide.lock.`

File: .travis.yml (hunk 1 above)
Comment (ahmetalpbalkan) [inline .travis.yml]: @garimakhulbe that's why we probably shouldn't have updated the `x/crypto` in glide.lock in the first place.

Comment (garimakhulbe) [inline .travis.yml]: @ahmetalpbalkan Then we need to manually update the commit number in glide.lock and revert go-autorest lock file. Shall I do that?

**Comment by garimakhulbe** [comment]
@ahmetalpbalkan As per our discussion, I am merging it for now and will remove this new dependency once go 1.7 is out.

**Pull request merged**